docs/source/configuration/internal-schemas.rst

```diff
@@ -0,0 +1,35 @@
+.. _internal-schema-configuration:
+
+=============================
+Internal Schema Configuration
+=============================
+
+This document covers configuration pertaining to settings about the internal schema that AtlasDB uses for storing
+some of its state. These settings should be specified under the ``internalSchema`` key in an
+``AtlasDbRuntimeConfiguration``.
+
+For a full list of the configurations available at this block, see
+`InternalSchemaConfig.java <https://github.com/palantir/atlasdb/blob/develop/atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/internalschema/InternalSchemaConfig.java>`__.
+
+Target Transactions Schema Version
+==================================
+
+.. warning::
+
+   _transactions2 is currently only supported for Cassandra and In-Memory KVSes.
+   In the case of Cassandra, _transactions2 is also only supported if you are using a version of Cassandra that supports
```

Comment:
If you don't have multi-pue, your Cassandra KVS is not going to be able to commit in `_transactions(1)` either, so seems like a superfluous comment

Reply:
Good point, didn't think about this. Will remove this line.